src/main/java/org/roy/loadx/Util.java

```diff
@@ -3,28 +3,18 @@
 public class Util {
 	
 	public static String findBoundaries(String leftBoundary, String rightBoundary, String str) {
```

Comment:
parameter name is "str" not source should be referenced as such in rest of code, could be changed to source maybe but probably not worth it